build-logic/settings.gradle.kts

```diff
@@ -3,8 +3,9 @@ rootProject.name = "build-logic"
 dependencyResolutionManagement {
     repositories {
         google()
-        gradlePluginPortal()
         mavenCentral()
+        gradlePluginPortal()
```

Comment:
I got this similar feedback on a PR of mine recently which I agree with. The ordering should be:
gradlePluginPortal()
mavenCentral()
Google()

And we actually can remove jerbrain compose dev url, it's available via mavenCentral() already